EIPS/eip-5976.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Rationale and Security Considerations were suggested to be removed by Editor. This will need a manual merge

Reply:
You should add this to:

https://github.com/ethereum/EIPs/blob/cf35f4c7097dd12618a741c5eb69199e0c72e6df/.github/workflows/ci.yml#L119